vendor.conf

```diff
@@ -4,17 +4,25 @@ github.com/coreos/etcd 824277cb3a577a0e8c829ca9ec557b973fe06d20
 github.com/cpuguy83/go-md2man a65d4d2de4d5f7c74868dfa9b202a3c8be315aaa
 github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76
 github.com/docker/distribution edc3ab29cdff8694dd6feb85cfeb4b5f1b38ed9c
-github.com/docker/docker 84144a8c66c1bb2af8fa997288f51ef2719971b4
 github.com/docker/docker-credential-helpers 3c90bd29a46b943b2a9842987b58fb91a7c1819b
 
+# docker/docker and dependencies
+github.com/docker/docker 14ce1f1cf48e9859223c6311de58aec4dc0f046c
+github.com/containerd/continuity 22694c680ee48fb8f50015b44618517e2bde77e8
+
+# TODO remove this once tonistiigi/fsutil switched, and was vendored (https://github.com/tonistiigi/fsutil/pull/15)
+github.com/stevvooe/continuity 22694c680ee48fb8f50015b44618517e2bde77e8
+github.com/moby/buildkit aaff9d591ef128560018433fe61beb802e149de8
+github.com/tonistiigi/fsutil 1dedf6e90084bd88c4c518a15e68a37ed1370203
```

Comment:
Let's keep this file sorted alphabetically instead of these adhoc groupings

Reply:
Ok, let me update 👍